### PR TITLE
Climb down

### LINKED ISSUE #62: Does player ever have to climb down?

### PR DESCRIPTION
**Pull request by BethMyre**
Player is now required to climb down from something after they've climbed up onto it.

### CODE REVIEW

File: adventure.go [hunk 1]
```diff
@@ -167,11 +170,12 @@ Goto:
 	rooms = nil
 	rooms = g.Rooms
 
-	inventory = nil
```
Comment (ewk) [inline adventure.go]: Why is this line deleted?

File: parser.go [hunk 2]
```diff
@@ -344,6 +344,8 @@ func climbStuff(feature string) {
 		fmt.Println("From on top of the dining room table you can get a better look at the candelabra.")
 		curRoom.Items["candelabra"].Discovered = true
 		fmt.Println("There's wax everywhere but it looks like there might still be a bit of candle left. Is that a candle? Look closer")
+	} else if feature == "down" {
+		climbedUp = false
```
Comment (ewk) [inline parser.go:351]: We should inform the player their state has changed, ie: 
`fmt.Println("You climb back down to the ground before you get dizzy!")`

File: adventure.go (hunk 1 above)
Comment (BethMyre) [reply adventure.go]: Good catch. Fixed.

File: parser.go (hunk 2 above)
Comment (BethMyre) [reply parser.go:351]: Done!
